week-3/27-attributes/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+npm r<!DOCTYPE html>
```

Comment:
Again, review your changes using `git diff` before committing to pick up mistakes like this.

Reply:
edeted